common/com/twitter/intellij/pants/model/TargetAddressInfo.java

```diff
@@ -28,10 +28,10 @@
    */
   private String pants_target_type = null;
 
-  private boolean is_code_gen;
+  private boolean is_synthetic;
 
-  public boolean isCodeGen() {
-    return is_code_gen;
+  public boolean isSynthetic() {
+    return is_synthetic;
```

Comment:
Does this generate a json parser? Where are we changing our JSON parsing to consume this new field?

Reply:
A gson parser is using certain conventions: https://github.com/pantsbuild/intellij-pants-plugin/blob/c3ae7eaf7f75ca2fea1194ccb9b78635b607974a/src/com/twitter/intellij/pants/service/project/model/TargetInfoDeserializer.java#L36 to variable name to the json field.

Reply:
Awesome, thank you!